src/sql/workbench/browser/modal/modal.ts

```diff
@@ -459,7 +456,7 @@ export abstract class Modal extends Disposable implements IThemable {
 	 * Shows the modal and attaches key listeners
 	 */
 	protected show() {
-		this.positionDialog();
+		this.positionCalloutDialog();
```

Comment:
if this is only applicable to callout dialog, we should check the style before calling the method.

Reply:
Fixed.